instrumentation/opentelemetry-instrumentation-boto3sqs/setup.cfg

```diff
@@ -56,4 +56,4 @@ where = src
 
 [options.entry_points]
 opentelemetry_instrumentor =
-    boto3sqs = opentelemetry.instrumentation.boto3sqs:Boto3SQSInstrumentation
+    boto3 = opentelemetry.instrumentation.boto3sqs:Boto3SQSInstrumentor
```

Comment:
Should we change the entry point name from `boto3sqs` to `boto3`?

Reply:
Yes. I missed that when creating the library, but there is no actual `boto3sqs` library, it's a sub-section of the `boto3` library. If we don't change it to `boto3` it will never be called